src/Admin/Controllers/HomeController.cs

```diff
@@ -60,25 +63,31 @@ public async Task<IActionResult> GetLatestDockerHubVersion(string repository, Ca
                     }
                 }
             }
-            catch (HttpRequestException) { }
+            catch (HttpRequestException e)
+            {
+                _logger.LogError(e, $"Error encountered while sending GET request to {requestUri}");
```

Comment:
I wonder if we could just throw a proper exception here which would be picked up in the logs, and return a 500 error which can be handled by the js.

Reply:
@bitwarden/dept-devops Could we potentially return the error message here directly or would that be potentially sensitive information? 

`return new JsonResult(e.Message) { StatusCode = StatusCodes.Status500InternalServerError };`

Reply:
It looks like we are using docker hub anonymously, so I don't think there is sensitive information that would be logged.

Reply:
@joseph-flinn Thanks, what about the request on line 79, to the web vault, could that result in an exception with a message that contains sensitive information?

@Hinton Are you thinking of logging the error **and** returning the 500 error, or do you think just returning the 500 error containing the error message is enough?

Reply:
@Manolachi Both I think. That way the logs will contain the errors in more details should they be needed.

Reply:
Done.

Reply:
Cheers, I'll get this reviewed shortly.